lighthouse/chain-config.yml

```diff
@@ -0,0 +1,63 @@
+CONFIG_NAME: "local"
+ALTAIR_FORK_EPOCH: 1
+BELLATRIX_FORK_EPOCH: 2
+EIP4844_FORK_EPOCH: 3 # ensure this happens at exactly shardingForkBlock in execution
+SECONDS_PER_SLOT: 6
+SLOTS_PER_EPOCH: 3
+TERMINAL_TOTAL_DIFFICULTY: 2
+
+# avoid collisions with mainnet/eip4844 config versioning
+GENESIS_FORK_VERSION: 0x00000ffe
+ALTAIR_FORK_VERSION: 0x01000ffe
+BELLATRIX_FORK_VERSION: 0x02000ffe
+EIP4844_FORK_VERSION: 0x03000ffe
+
+DOMAIN_BLOBS_SIDECAR: 0x0a000000
+
+# All missing values below (except for DEPOSIT_CONTRACT_ADDRESS) copied from mainnet 
```

Comment:
Hi,
Why aren't these configuration required for Prysm? Does Prysm populate the mainnet config by default?

Reply:
yup that's exactly it. We should make these explicit in the common beacon config so all clients can easily reuse them.

Reply:
Thanks @Inphi! I'll create a separate PR for the explicit beacon config , as this one will probably take a while to get merged

Reply:
https://github.com/Inphi/eip4844-interop/pull/48